quinn-proto/src/connection/mod.rs

```diff
@@ -3612,10 +3635,63 @@ pub enum Event {
     DatagramReceived,
 }
 
+#[derive(Copy, Clone)]
 struct PathResponse {
     /// The packet number the corresponding PATH_CHALLENGE was received in
     packet: u64,
     token: u64,
+    /// The address the corresponding PATH_CHALLENGE was received from
+    remote: SocketAddr,
+}
+
+#[derive(Default)]
```

Comment:
This should probably be above `PathResponse`?

Maybe move this and `PathResponse` into `paths` to avoid ballooning the size of `connection/mod.rs` even more?

Reply:
Done.